Packs/IntegrationsAndIncidentsHealthCheck/Scripts/GetFailedTasks/GetFailedTasks.py

```diff
@@ -1,10 +1,41 @@
 import demistomock as demisto  # noqa: F401
 from CommonServerPython import *  # noqa: F401
 
+BRAND = "Demisto REST API"
+
+
+def get_rest_api_instance_to_use(rest_api_instance_from_arg):
+    """
+        This function checks if there are more than one instance of demisto rest api.
+        Args:
+            rest_api_instance_from_arg (str) : rest api instance name
+
+        Returns:
+            Demisto Rest Api instance to use
+    """
+    allInstances = demisto.getModules()
+    number_of_rest_api_instances = 0
+    rest_api_instance_to_use = None
+    for instance_name in allInstances:
+        if allInstances[instance_name]['brand'] == BRAND:
+            rest_api_instance_to_use = instance_name
+            number_of_rest_api_instances += 1
+    if number_of_rest_api_instances > 1:
+        return_error("GetFailedTasks: This script can only run with a single instance of the Demisto REST API. "
+                     "Specify the instance name in the 'rest_api_instance' argument.")
+    elif number_of_rest_api_instances == 1:
+        return rest_api_instance_to_use
+    else:
+        # in there are no rest api instances
+        return rest_api_instance_from_arg
```

Comment:
this should be the default, not the else @abaumgarten

Reply:
No problem, changed.